<launch>
    <node name="pointcloud_saver" type="pointcloud_saver" pkg="pcl_slam_handler" />
</launch>